src/rule_dsl.jl

```diff
@@ -185,8 +185,8 @@ getdepth(r::ACRule) = getdepth(r.rule)
 macro acrule(expr)
     arity = length(expr.args[2].args[2:end])
     quote
-        ACRule(@rule($expr), $arity)
-    end
+        SymbolicUtils.ACRule(SymbolicUtils.@rule($(expr)), $arity)
```

Comment:
```suggestion
        $(@__MODULE__).ACRule($(@__MODULE__).@rule($(expr)), $arity)
```
If I call this from Main, and SymbolicUtils is not loaded (e.g. when using ModelingToolkit) then explicitly referencing `SymbolicUtils` might fail.

Reply:
thanks!